formats/mainstream_browse_page/publisher/details.json

```diff
@@ -3,6 +3,9 @@
   "type": "object",
   "additionalProperties": false,
   "properties": {
+    "child_ordering": {
+      "type": "string"
```

Comment:
We have already a validation in place within `collections-publisher` and I thought that was enough. But doing what you suggested means future-proofing in case of some other apps will publish to `content-store`. Thanks, will do.